src/main/kotlin/com/hypto/iam/server/service/PolicyService.kt

```diff
@@ -131,6 +147,60 @@ class PolicyServiceImpl : KoinComponent, PolicyService {
         val newContext = PaginationContext.from(policies.lastOrNull()?.hrn, context)
         return PolicyPaginatedResponse(policies.map { Policy.from(it) }, newContext.nextToken, newContext.toOptions())
     }
+
+    override suspend fun createPolicyFromTemplate(
+        organizationId: String,
+        request: CreatePolicyFromTemplateRequest,
+    ): Policy {
+        val accountId =
+            when (request.allAccountsAccess) {
+                true -> ".*"
+                else -> request.accountId
+            }
```

Comment:
IAM must not be aware of this logic. `allAccountsAccess` must be removed from API payload. There can be 2 templates. one with .* and other with {{account_id}} as template variable